lib/forms-schema.js

```diff
@@ -739,12 +739,24 @@ const SubmissionEventsSchema = Joi.object().keys({
         url: Joi.string()
           .required()
           .uri(),
-        container: Joi.string().required(),
+        container: Joi.object().keys({
+          uri: Joi.string().required(),
```

Comment:
```suggestion
          uri: Joi.number().required(),
```

Check with @kizaonline  but I think this is suppose to be a number like the others.

Reply:
Yes, all the Uri values are numeric